--- release.py	2017-10-05 23:08:45.464292800 +0200
+++ release_new.py	2018-03-16 07:14:56.432975200 +0100
@@ -12,7 +12,7 @@
 # properly comparable using normal operarors, for example:
 #  (6,1,0,'beta',0) < (6,1,0,'candidate',1) < (6,1,0,'candidate',2)
 #  (6,1,0,'candidate',2) < (6,1,0,'final',0) < (6,1,2,'final',0)
-version_info = (11, 0, 0, FINAL, 0, '')
+version_info = (11, 0, 0, FINAL, 0, ' %%CONTAINER_DATE%%-%%COMMIT_HASH%%')
 version = '.'.join(str(s) for s in version_info[:2]) + RELEASE_LEVELS_DISPLAY[version_info[3]] + str(version_info[4] or '') + version_info[5]
 series = serie = major_version = '.'.join(str(s) for s in version_info[:2])
 
